src/main/java/com/uber/cadence/internal/metrics/MetricsType.java

```diff
@@ -134,5 +134,8 @@
 
   public static final String STICKY_CACHE_HIT = CADENCE_METRICS_PREFIX + "sticky-cache-hit";
   public static final String STICKY_CACHE_MISS = CADENCE_METRICS_PREFIX + "sticky-cache-miss";
+  public static final String STICKY_CACHE_FORCED_EVICTION =
+      CADENCE_METRICS_PREFIX + "sticky-cache-forced-eviction";
```

Comment:
I think we want two metrics. One for thread forced another for count forced. May be one total another thread only.

Reply:
added one specifically for threads and the existing one will count the total